src/main/java/core/basesyntax/dao/ReportWriterImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.FruitTransactionParser;
+import core.basesyntax.model.Storage;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ReportWriterImpl implements ReportWriter {
+    private static FruitTransactionParser fruitTransactionParser = new FruitTransactionParser();
+    private Storage storage = new Storage();
+
+    @Override
+    public String generateReport(String fileName) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName))) {
+            String textInReport = "fruit,quantity\n"
```

Comment:
use `System.lineSeparator()` instead of `\n`

Maybe StringBuilder will be better to create report